charts/cinder-csi-plugin/values.yaml

```diff
@@ -70,10 +70,49 @@ secret:
 #      ca-file=/etc/cacert/ca-bundle.crt
 
 storageClass:
-  enabled: true
+  enabled: false
   delete:
```

Comment:
is this change plans to disable the default storageclass? seems Helm chart is helpful for deploy and tets
remove this from true to false means we lost this bydefault? 

I Think anyway we can manually edit this yaml for complicated case?

Reply:
Yes. our cloud provider offers [fixed volume_types](https://docs.otc.t-systems.com/api/evs/evs_04_2065.html) for Cinder volumes where we not able to create our own volume types to pass the default storage class provided by this Helm chart.

Reply:
@eumel8 Could we keep default as enabled: true as they work for most of the cases right.
If custom required , could be changed false and uncomment below staorageclass as per requirement?

Reply:
Agree with @ramineni